draft-ietf-quic-qpack.md

```diff
@@ -1253,51 +1253,51 @@ different web origins {{?RFC6454}} are made by mutually distrustful entities.
 
 ## Mitigation
 
-Users of HTTP that require confidentiality for header fields can use values with
-entropy sufficient to make guessing infeasible. However, this is impractical as
-a general solution because it forces all users of HTTP to take steps to mitigate
-attacks. It would impose new constraints on how HTTP is used.
+Users of HTTP that require confidentiality for header or trailer fields can use
+values with entropy sufficient to make guessing infeasible. However, this is
+impractical as a general solution because it forces all users of HTTP to take
+steps to mitigate attacks. It would impose new constraints on how HTTP is used.
 
 Rather than impose constraints on users of HTTP, an implementation of QPACK can
 instead constrain how compression is applied in order to limit the potential for
 dynamic table probing.
 
 An ideal solution segregates access to the dynamic table based on the entity
-that is constructing header fields. Header field values that are added to the
-table are attributed to an entity, and only the entity that created a particular
-value can extract that value.
+that is constructing the message. Field values that are added to the table are
+attributed to an entity, and only the entity that created a particular value can
+extract that value.
 
 To improve compression performance of this option, certain entries might be
 tagged as being public. For example, a web browser might make the values of the
 Accept-Encoding header field available in all requests.
 
-An encoder without good knowledge of the provenance of header fields might
-instead introduce a penalty for a header field with many different values, such
-that a large number of attempts to guess a header field value results in the
-header field not being compared to the dynamic table entries in future messages,
-effectively preventing further guesses.
+An encoder without good knowledge of the provenance of field values might
+instead introduce a penalty for a field name with many different values, such
```

Comment:
is this correct, or a typo?

Reply:
Shorthand, I think.  What it's trying to express is that if there are many field lines in the dynamic table which have the same field name and distinct values, you might with some probability ignore the dynamic table when encoding.  The more entries, the higher the probability you should ignore them all to frustrate an attacker.